src/Sensors.cpp

```diff
@@ -593,6 +654,21 @@ void Sensors::sht31Init() {
     sht31.begin(0x44);  // temp/humidity sensor
 }
 
+void Sensors::scd30Init() {
+    DEBUG("-->[SCD30] starting SCD30 sensor..");
+  if (scd30.begin() == true) {
+//OPTIONAL configuration
+  //Choose Measurement Interval 
+   //scd30.setMeasurementInterval(2); //Change number of seconds between measurements: 2 to 1800 (30 minutes)
+  //Sealevel
+   //scd30.setAltitudeCompensation(2600); //Set altitude of the sensor in m
+  //Pressure in Boulder, CO is 24.65inHg or 834.74mBar
+   //scd30.setAmbientPressure(1012); //Current ambient pressure in mBar: 700 to 1200
+  //Temperature offset
+   //scd30.setTemperatureOffset(5); //Optionally we can set temperature offset to 5°C
```

Comment:
if this comments aren't used, please remove it. I could exposed the sdc30 object public, for example the users could using it.

Reply:
is the option for configurate the SCD30, is used few times.
Is optional that is the reason for comments

Reply:
If we will using it in the future, we should implement it or pass the complete object sdc30 like public, but in master code we don't need this comments for now.

Reply:
done